src/Test.UnitTests.Sarif.Multitool.Library/LogResultTests.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    public class LogResultTests
+    {
+        // A simple schema against which a SARIF log file successfully validates.
+        // This way, we don't have to read the SARIF schema from disk to run these tests.
+        private const string SchemaFileContents =
+@"{
+  ""$schema"": ""http://json-schema.org/draft-04/schema#"",
+  ""type"": ""object""
+}";
+
+        private const string SchemaFilePath = @"c:\schemas\SimpleSchemaForTest.json";
+        private const string LogFileDirectory = @"C:\Users\John\logs";
```

Comment:
change to path that will work in both Windows and Linux like
 Path.Combine(Environment.CurrentDirectory, "schemas", "SimpleSchemaForTest.json");

 Path.Combine(Environment.CurrentDirectory, "John", "logs");

Reply:
These are const properties so they can't bet set by a method call. We already have this pattern working in other tests (I copy/pasted those).